README.md

```diff
@@ -26,9 +26,9 @@ tool. Check out our app [here!](https://algosort.netlify.app/)
 
 <br/>
 
-<p align="center"><strong>Radix Sort preview</strong></p>
+<p align="center"><strong>Bucket Sort preview</strong></p>
 
-<img src="images/RadixSort.gif" alt="" />
+<img src="images/BucketSort.gif" alt="" />
```

Comment:
The bucket sort gif looks like it is using the old colors before my PR was merged? Should update to reflect the new one.